packages/picker/test/index.ts

```diff
@@ -1150,50 +1167,42 @@ export function runPickerTests(): void {
                 expect(document.activeElement, 'focuses input 2').to.equal(
                     input2
                 );
-                // tab to the picker
                 let focused = oneEvent(el, 'focus');
-                await sendKeys({ press: 'Shift+Tab' });
-                await focused;
-
-                expect(el.focused, 'focused').to.be.true;
-                expect(el.open, 'closed').to.be.false;
-                expect(document.activeElement, 'focuses el').to.equal(el);
+                if (!isSafari) {
```

Comment:
Safari should only have Tab focusable elements if the user preferences are configured to do so. We should not force something contrary to user preferences.